agent/structs/errors.go

```diff
@@ -65,6 +67,10 @@ func IsErrUsingV2CatalogExperiment(err error) bool {
 	return err != nil && strings.Contains(err.Error(), errUsingV2CatalogExperiment)
 }
 
+func IsErrSamenessGroupNotFound(err error) bool {
+	return err != nil && strings.Contains(err.Error(), errSamenessGroupNotFound)
+}
+
```

Comment:
No concern having this in CE given this is not CE-exclusive code, but seems net-new. I'm guessing it was left out in past Ent work.

Reply:
yes.  i had to fix this.  it should have been in CE just like `IsErrSamenessGroupMustBeDefaultForFailover`